server/src/main/java/org/elasticsearch/index/get/ShardGetService.java

```diff
@@ -201,9 +202,14 @@ private GetResult innerGetLoadFromStoredFields(String type, String id, String[]
 
             if (!fieldVisitor.fields().isEmpty()) {
                 fieldVisitor.postProcess(mapperService);
-                fields = new HashMap<>(fieldVisitor.fields().size());
+                nonMetaDataFields = new HashMap<>();
```

Comment:
we should use `documentFields` for consistency

Reply:
done